src/go/rpk/pkg/utils/args.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package utils
```

Comment:
There's a [`cli`](https://github.com/redpanda-data/redpanda/tree/dev/src/go/rpk/pkg/cli) package where I think this file would fit perfectly, instead of the general top-level utils package.

Reply:
Moved to `cli` package. Thanks